DESCRIPTION

```diff
@@ -89,7 +90,9 @@ Remotes:
     MoTrPAC/MotrpacRatTraining6moData,
     MoTrPAC/MotrpacBicQC,
     YuLab-SMU/ggtree,
-    nicolerg/ssGSEA2
+    nicolerg/ssGSEA2,
+    cytoscape/RCy3,
+    plotrix/plotrix
```

Comment:
download `plotrix` from GitHub to avoid `R-CMD check` warning "Requires (indirectly) orphaned package: ‘plotrix’"

Reply:
`plotrix` is a dependency for `mutoss`, which is a dependency for `metap`